src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -4,10 +4,7 @@
  */
 
 .bd-sidebar-primary {
-  display: flex;
-  flex-direction: column;
-  gap: 1rem;
-
+  flex-shrink: 0;
```

Comment:
what does this do?

and what was the reason for removing the flex/flex-direction/etc stuff above?

Reply:
I will revert this one. The change does not handle all cases, like, when there is an ethical ads banner. It does not shift the ethical ads banner to the bottom of the primary sidebar, when the toc links do not occupy the whole screen. 

For example, it produces this: 
<img width="422" alt="Screen Shot 2022-08-12 at 1 15 03 pm" src="https://user-images.githubusercontent.com/6542997/184278563-ea8998e6-04a3-403d-8fe8-b23fd0bc6851.png">


Instead of this: 

<img width="445" alt="Screen Shot 2022-08-12 at 1 15 18 pm" src="https://user-images.githubusercontent.com/6542997/184278592-3e8e29e7-f044-4de1-971e-b4cb846358d2.png">


Thanks for pointing this change out.